debian/jessie/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian:jessie
+
+LABEL maintainer me@danvaida.com
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update -y && \
+    apt-get install --fix-missing && \
+    apt-get install -y gcc='4:4.9.2-2' \
+                       libffi-dev='3.1-2+b2' \
+                       libssl-dev='1.0.1t-1+deb8u6' \
+                       python=2.7.9-1 \
+                       python-dev=2.7.9-1 \
+                       python-pip=1.5.6-5 \
+                       python-yaml=3.11-2
+
+RUN apt-get remove -y --purge python-cffi
```

Comment:
Removals should be run in the same command that add them, otherwise they have no effect to optimize the image. So, this command should be appended to the previous one to reduce the image size.